### PR TITLE
fix: add pythonhubdev

### LINKED ISSUE #1467: Vetrichelvan
## Profile Screenshot

Screenshot 2022-10-05 at 8 02 03 PM

### PR DESCRIPTION
**Pull request by pythonhubdev**
resolves #1467 

## Checklist for a profile addition

- [x] Name the PR `fix: add pythonhubdev`.
- [x] You've added file named `pythonhubdev.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

### CODE REVIEW

File: profiles/pythonhubdev.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/60695664/194086997-2673cbdd-f7ad-4ba3-8980-ec2500de3fe8.png",
+  "issueId": 1467,
+  "name": "Vetrichelvan",
+  "username": "pythonhubdev"
+}
```
Comment (Panquesito7) [inline profiles/pythonhubdev.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by pythonhubdev** [comment]
@Panquesito7 I have committed the changes

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by pythonhubdev** [comment]
@Panquesito7 will this PR be merged?

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**